actions/add_update_repository_team.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: add_update_repository_team
+runner_type: python-script
+pack: github
+description: >
+  Add or update repository team.
+  Example:
+    st2 run github.add_update_repository_team  organization="organization" owner="owner" repo="reponame" team_slug="team_id" api_user="token_name"
```

Comment:
This example uses a parameter called organization, but the yaml states that the parameter is called org.